src/fixes/duplicated_key.rs

```diff
@@ -0,0 +1,152 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct DuplicatedKeyFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for DuplicatedKeyFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "DuplicatedKey",
+        }
+    }
+}
+
+impl Fix for DuplicatedKeyFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_warnings(
+        &self,
+        warnings: Vec<&mut Warning>,
+        lines: &mut Vec<LineEntry>,
+    ) -> Option<usize> {
+        let len = warnings.len();
```

Comment:
can we define this closer to where it's used (or maybe just not declare it at all, ```Some(warnings.len())``` seems readable too